openpype/hooks/pre_non_python_host_launch.py

```diff
@@ -51,3 +51,7 @@ def execute(self):
 
         if remainders:
             self.launch_context.launch_args.extend(remainders)
+
+        self.launch_context.kwargs = \
+            get_non_python_host_kwargs(self.launch_context.kwargs)
```

Comment:
Logic from `get_non_python_host_kwargs` is used only here. I would not move it to lib. You are moving function for single use from almost empty file to overfilled file...